src/components/blocks/section-header/section-header.html

```diff
@@ -1,3 +1,15 @@
 <header class="b-section-header{% for m in modifiers %} b-section-header--{{ m }}{% endfor %}">
+  {% if modifiers and 'alternate-image' in modifiers %}
+    <img alt=""
+      class="b-section-header__media-image"
+      itemprop="image"
+      sizes="(max-width: 499px) calc(100vw - 20px),
+            (min-width: 500px) calc(50vw - 40px),
+            (min-width: 992px) calc(33vw - 80px)"
```

Comment:
Thanks, I presume you meant "sizes" rather than "size" even though there is only one but this is the attribute name, I believe.  have changed this to `sizes="100vw"`

Reply:
absolutely correct! my typo, sorry

Reply:
This is now addressed.